backend/npdfhir/mappings.py

```diff
@@ -17,6 +17,12 @@ def toNPD(self, fhirValue):
         else:
             return self.mapping.inverse[fhirValue]
 
+    def keys(self, which='fhir'):
+        if which == 'npd':
+            return list(self.mapping.keys())
+        else:
+            return list(self.mapping.inverse.keys())
```

Comment:
I'm expecting to see `keys` get called with the `which` argument somewhere in this PR or in existing code, but didn't see it in GitHub search. Tell me more about this?